src/site_assessment/best_fit_polygons.jl

```diff
@@ -123,13 +123,14 @@ function identify_potential_sites_edges(
     n_rot_p_side::Int64=2,
     surr_threshold::Float64=0.33
 )::DataFrame
-    reef_lines = reef_lines[gdf.management_area .== region]
-    gdf = gdf[gdf.management_area .== region, :]
+    region_long = REGIONAL_DATA["region_long_names"][region]
+    reef_lines = reef_lines[gdf.management_area .== region_long]
+    gdf = gdf[gdf.management_area .== region_long, :]
     max_count = (
-        (x_dist / degrees_to_meters(res, mean(indices_pixels.dims[2]))) *
+        (x_dist / degrees_to_meters(res, mean(search_pixels.lat))) *
```

Comment:
working with x, should this be `search_pixels.lon`?

Also repeating expressions. extract constants?

Reply:
I think `lat` is intended here. Rasters.jl treats matrices as being long/lat (X/Y, X being the first dimension)

Reply:
No wait, I think you're right, something strange here...